### PR TITLE
feat (association-select): hiding and enable/disable of placeholder option, as well as custom text

### PR DESCRIPTION
**Pull request by go4cas**
Provide the developer with options to show/hide and enable/disable the placeholder option
, as well as overriding the default display text value.

Backwards compatible, and fixing issues #169 and #170

### CODE REVIEW

File: src/component/association-select.js [hunk 1]
```diff
@@ -28,6 +28,12 @@ export class AssociationSelect {
 
   @bindable multiple = false;
 
+  @bindable hidePlaceholder = false;
+
+  @bindable allowPlaceholderSelect = false;
```
Comment (doktordirk) [inline src/component/association-select.js]: how about 'selectablePlaceholder' (my preference) or 'isPlaceholderSelectable'

File: src/component/association-select.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <template>
   <select class="form-control" value.bind="value" multiple.bind="multiple">
-    <option selected disabled value="0">- Select a value -</option>
+    <option selected disabled.bind="!hidePlaceholder && !allowPlaceholderSelect" value="0" show.bind="!hidePlaceholder">${placeholderText || '- Select a value -'}</option>
```
Comment (doktordirk) [inline src/component/association-select.html]: has 'disabled.bind="!hidePlaceholder'  any effect?  hmm guess so.

**Comment by doktordirk** [comment]
well, concise and user-friendly. i like it

**Comment by doktordirk** [comment]
you accidentally commited the package.json now

**Comment by go4cas** [comment]
Uhm, how do I undo that?

**Comment by doktordirk** [comment]
depends on your tool of choice. but you can just commit the previous version on top of this

**Comment by doktordirk** [comment]
hmm, that didn't work well. now it's back at allowPlaceholderSelect 
try:
git reset --hard HEAD~2

change again allowPlaceholderSelect to selectablePlaceholder

git add -A
git commit --amend --no-edit
git push origin HEAD -f

and then if you're nice you'll update /doc/component.md :smile:

File: doc/components.md [hunk 3]
```diff
@@ -25,6 +25,43 @@ Get all entries of the view-models 'userRepository' repository property and popu
 ></association-select>
 ```
 
+## Basic example, with placeholder settings
+
+The same as the basic example above, with:
+
+* hiding the placeholder option (`value=0`):
```
Comment (doktordirk) [inline doc/components.md]: value=0 associates someone there is a value=0 to set. maybe better value===0

**Comment by doktordirk** [comment]
ah that worked
now you can just add changes
so eg when chaining the readme commit with
doc(README): xyz

no need to use the old message again, since that would already be logged in the changelog

File: doc/components.md (hunk 3 above)
Comment (doktordirk) [inline doc/components.md]: though thinking of it. maybe just mention once or twice at the right place that placeholder means value 0 and then just use placeholder. that'll make it more readable

**Comment by doktordirk** [comment]
alright. tx: nice job!! let's get that in then

**Pull request merged**